web-app/js/portal/details/SubsetPanel.js

```diff
@@ -18,7 +18,6 @@ Portal.details.SubsetPanel = Ext.extend(Ext.Container, {
 
         var config = Ext.apply({
             title: OpenLayers.i18n('subsetPanelTitle'),
-            hideMode: 'offsets', // fixes #1278
```

Comment:
why do we need to delete hide mode here? Just want to check that this issue won't reapear